resources/r/model.R

```diff
@@ -60,7 +60,20 @@ manipulate_regions_for_shapefile <- function(region_prevalence) {
   return(thirteen_level_NHS_regional_prevalence)
 }
   
+
+#join shapefile to regional prevalence data
+join_difference_to_shapefile <- function(thirteen_level_NHS_regional_prevalence,region_shapefile){
```

Comment:
function name doesn't fit what it is doing, or the flowchart - please could you change it thanks!